devtools/ci.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e -x
+
+isort --check --diff crate/ tests/ setup.py
+flake8 crate/crash
+coverage run -m unittest -v
```

Comment:
Thank you. Maybe also call this new helper program directly from the GHA recipe, in order not to duplicate the code?

https://github.com/crate/crash/blob/ffb80607af2ef89d83388210ba91dd37e720ab6f/.github/workflows/main.yml#L49-L53